.github/workflows/publish.yaml

```diff
@@ -0,0 +1,54 @@
+name : Publish Webhook Images
+
+on:
+  push:
+    tags:
+      - release/v*
```

Comment:
I don't think it needs to be quoted because the string value doesn't start with a `*`:
```
$ pbpaste | yq .
{
  "on": {
    "push": {
      "tags": [
        "release/v*"
      ]
    }
  }
}
```